docker/bin/docker_build.sh

```diff
@@ -40,7 +40,11 @@ DOCKER_IMAGE_TAG="${DOCKER_REPO}/bedrock_${DOCKERFILE}:${DOCKER_TAG}"
 
 # generate the dockerfile
 rm -f "$FINAL_DOCKERFILE"
-cat "docker/dockerfiles/bedrock_$DOCKERFILE" | envsubst '$GIT_COMMIT' > "$FINAL_DOCKERFILE"
+cat "docker/dockerfiles/bedrock_$DOCKERFILE" | envsubst '$GIT_COMMIT,$BRANCH_NAME' > "$FINAL_DOCKERFILE"
```

Comment:
I don't believe we need envsubst any more: https://docs.docker.com/engine/reference/builder/#environment-replacement

Reply:
Our docker on Jenkins is too old for that to work. Will fix many things when we can use new docker.

Reply:
More accurately it doesn't work on the `FROM` line in our version.